functions/openhabian.bash

```diff
@@ -117,14 +117,6 @@ system_check_default_password() {
   fi
 }
 
-openhabian_hotfix() {
-  if ! grep -q "sleep" /etc/cron.d/firemotd; then
-    introtext="It was brought to our attention that openHABian systems cause requests spikes on remote package update servers. This unwanted behavior is related to a simple cronjob configuration mistake and the fact that the openHABian user base has grown quite big over the last couple of months. Please continue to apply the appropriate modification to your system. Thank you."
-    if ! (whiptail --title "openHABian Hotfix Needed" --yes-button "Continue" --no-button "Cancel" --yesno "$introtext" 15 80) then return 0; fi
-    firemotd
-  fi
-}
-
```

Comment:
This brings back great memories 😃 
Imagine you start off with a hobby project to help some people by automating repetitive setup tasks. Your project becomes bigger and bigger and before you know it you are contacted by an international package distributor because your "bots" create outages on their server 😅 

Not needed anymore. Thanks!

Reply:
😃